apps/component-date/fields/index.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+const dateComponent = require('hof-component-date');
+
+module.exports = {
+  'date-of-birth': dateComponent('date-of-birth', {
+    // before means a date cannot be in the future
+  }),
+  'passport-expiry': dateComponent('passport-expiry', {
+    // after means a date cannot be in the past
+    validate: ['required', 'after']
```

Comment:
Ugh. I just looked, and it seems that `before` and `after` take different arguments, which sucks spectacular balls. 

https://github.com/UKHomeOfficeForms/hof-form-controller/blob/master/lib/validation/validators.js#L88-L118